src/content/en/fundamentals/performance/optimizing-content-efficiency/automating-image-optimization/index.md

```diff
@@ -0,0 +1,1615 @@
+
+
+<img
+        src="https://res.cloudinary.com/ddxwdqwkr/image/upload/v1502426282/essential-image-optimization/image-optimisation"
+        alt="Image optimization covers a number of different techniques" />
+<strong>Image optimization:</strong> Choose the right format, compress carefully and prioritize critical images over those that can be lazy-loaded.
+
+
+Common image optimizations include compression, responsively serving them down based on screen size using [`<picture>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture)/[`<img srcset>`](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images), and resizing them to reduce image decode costs.
+
+
+<img
+       src="https://res.cloudinary.com/ddxwdqwkr/image/upload/v1502834117/chart_naedwl.jpg"
+        alt="A histogram of potential image savings from the HTTP Archive validating the 30KB of potential image savings at the 95th percentile." />
+Per the [HTTP Archive](http://jsfiddle.net/rviscomi/rzneberp/embedded/result/), per-image savings at the 95th percentile (looking at the Cumulative Distribution Function) are 30KB!</strong>
+
+
+There's plenty of room for us to collectively optimize images better.
+
+
+<img
+        src="https://res.cloudinary.com/ddxwdqwkr/image/upload/v1502519576/essential-image-optimization/image-optim.jpg"
+        alt="ImageOptim in use on Mac with a number of images that have been compressed with savings over 50%" />
+
+ImageOptim is free, reduces image size through modern compression techniques and by stripping unnecessary EXIF meta-data.
+
+
+
+If you're a designer, there's also an [ImageOptim plugin for Sketch](https://github.com/ImageOptim/Sketch-plugin) that will optimize your assets on export. I've found it a huge time saver.
+
+### <a id="do-my-images-need-optimization" href="#do-my-images-need-optimization">How can I tell if my images need to be optimized?</a>
+
+Perform a site audit through [WebPageTest.org](https://www.webpagetest.org/) and it will highlight opportunities to better optimize your images (see "Compress Images").
+
+
+<img
+        src="images/Modern-Image1.jpg"
+        alt="WebPage test supports auditing for image compression via the compress images section" />
+
+The "Compress Images" section of a WebPageTest report lists images that can be compressed more efficiently and the estimated file-size savings of doing so.
+
+
+
+
+
+<img
+        src="images/Modern-Image2.jpg"
+        alt="image compression recommendations from webpagetest" />
+
+
+
+[Lighthouse](https://developers.google.com/web/tools/lighthouse/) audits for performance best practices. It includes audits for image optimisation and can make suggestions for images that could be compressed further or point out images that are off-screen and could be lazy-loaded.
```

Comment:
Make internal links absolute - ie `/web/tools/lighthouse/`. We need to exclude the domain because in some cases we're served on a different domain, for example `developers.google.cn` so that it works in China.